tests/framework/integration/cluster.go

```diff
@@ -743,14 +747,23 @@ func MustNewMember(t testutil.TB, mcfg MemberConfig) *Member {
 	return m
 }
 
-func memberLogger(t testutil.TB, name string) *zap.Logger {
+func memberLogger(t testutil.TB, name string) (*zap.Logger, *testutils.LogObserver) {
 	level := zapcore.InfoLevel
 	if os.Getenv("CLUSTER_DEBUG") != "" {
 		level = zapcore.DebugLevel
 	}
 
-	options := zaptest.WrapOptions(zap.Fields(zap.String("member", name)))
-	return zaptest.NewLogger(t, zaptest.Level(level), options).Named(name)
+	obCore, logOb := testutils.NewLogObserver(level)
+
+	options := zaptest.WrapOptions(
+		zap.Fields(zap.String("member", name)),
+
+		// copy logged entities to log observer
+		zap.WrapCore(func(oldCore zapcore.Core) zapcore.Core {
+			return zapcore.NewTee(oldCore, obCore)
+		}),
+	)
+	return zaptest.NewLogger(t, zaptest.Level(level), options).Named(name), logOb
```

Comment:
Yeah. First version is to disable observer by default. I was concerned about memory usage because the github action node nanny might send the sigkill randomly. I think it can be good to be optional item.

Reply:
I retest it and it doesn't bring too many overhead. So I keep it as it is.